lib/sinatra/base.rb

```diff
@@ -1947,6 +1952,7 @@ def self.delegate(*methods)
           return super(*args, &block) if respond_to? method_name
           Delegator.target.send(method_name, *args, &block)
         end
+        ruby2_keywords method_name
```

Comment:
We can also avoid needing to use the `ruby2_keywords` gem if we do

```suggestion
        ruby2_keywords(method_name) if respond_to?(:ruby2_keywords, true)
```